### PR TITLE
Fix for lighting on empty background

### PR DESCRIPTION
**Pull request by unity-cchu**
### Checklist for PR maker
- [x] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.

---
### Purpose of this PR
This is a fix for an issue where transparency with normal maps on an empty background resulted in incorrect lighting values.

---
### Testing status

**Manual Tests**: What did you do?
- [X] Opened test project + Run graphic tests locally
- [X] Checked against Lost Crypt project

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

### CODE REVIEW

File: com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl [hunk 1]
```diff
@@ -26,10 +26,12 @@
             output.screenUV = ComputeScreenPos(clipVertex).xy; \
             output.positionWS = worldSpacePos;
 
+//half3 normalUnpacked = UnpackNormal(normal);
         #define APPLY_NORMALS_LIGHTING(input, lightColor)\
             half4 normal = SAMPLE_TEXTURE2D(_NormalMap, sampler_NormalMap, input.screenUV);\
-            half3 normalUnpacked = UnpackNormal(normal);\
+            half3 normalUnpacked;\
             half3 dirToLight;\
+            normalUnpacked.xyz = 2 * normal.rgb - 1;\
```
Comment (rainsing) [inline com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl:34]: We should use UnpackNormalRGBNoScale(), which does exactly the same calculation.

File: com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl [hunk 2]
```diff
@@ -26,10 +26,12 @@
             output.screenUV = ComputeScreenPos(clipVertex).xy; \
             output.positionWS = worldSpacePos;
 
+//half3 normalUnpacked = UnpackNormal(normal);
```
Comment (rainsing) [inline com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl:29]: Remove this?

File: com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl (hunk 1 above)
Comment (rainsing) [reply com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl:34]: Also, the same change should be applied to the other branch of the #ifdef.

**Review by rainsing — APPROVED** [review]

File: com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl (hunk 1 above)
Comment (unity-cchu) [reply com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl]: Fixed

File: com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl (hunk 2 above)
Comment (unity-cchu) [reply com.unity.render-pipelines.universal/Shaders/2D/Include/LightingUtility.hlsl]: Fixed

**Review by rainsing — APPROVED** [review]

**Review by darrenquek — APPROVED** [review]
FogBugz Case tracked here:
https://fogbugz.unity3d.com/f/cases/1276992/

Fix Verified on:
Version: 2020.2.0b2.3094 Personal
Revision: trunk 90b2443a8df1
Built: Tue, 01 Sep 2020 08:25:37 GMT

**Pull request merged**